pkg/cmd/project/variables/view/view.go

```diff
@@ -113,55 +140,148 @@ func viewRun(opts *ViewOptions) error {
 		return fmt.Errorf("cannot find variable '%s'", opts.name)
 	}
 
-	fmt.Fprintln(opts.out, output.Bold(filteredVars[0].Name))
+	outputFormat, err := opts.cmd.Flags().GetString(constants.FlagOutputFormat)
 
-	for _, v := range filteredVars {
-		data := []*output.DataRow{}
+	if err != nil { // should never happen, but fallback if it does
+		outputFormat = constants.OutputFormatTable
+	}
 
-		data = append(data, output.NewDataRow("Id", output.Dim(v.GetID())))
-		if v.IsSensitive {
-			data = append(data, output.NewDataRow("Value", output.Bold("*** (sensitive)")))
-		} else {
-			data = append(data, output.NewDataRow("Value", output.Bold(v.Value)))
-		}
+	out := opts.cmd.OutOrStdout()
 
-		if v.Description == "" {
-			v.Description = constants.NoDescription
-		}
-		data = append(data, output.NewDataRow("Description", output.Dim(v.Description)))
+	switch strings.ToLower(outputFormat) {
+	case constants.OutputFormatBasic, constants.OutputFormatTable:
+		fmt.Fprintln(out, output.Bold(filteredVars[0].Name))
+		for _, v := range filteredVars {
+			data := []*output.DataRow{}
 
```

Comment:
Should this also print out the `KEY    VALUE` heading like the project view does?
```suggestion
data = append(data, output.NewDataRow(output.Bold("KEY"), output.Bold("VALUE")))
```

Reply:
Not against that change - although it perhaps should _only_ apply to the **table** format (as it is with `project view`) ?

Reply:
Yeah definitely, it would probably be a bit confusing in the basic format

Reply:
I've made a change in https://github.com/OctopusDeploy/cli/pull/328/commits/98078bf4968bee4e7536552ce1577ddd5a25b7e9 but wanted to mention that Ben P mentioned he was going to review the `view` command output format holistically. See [this thread](https://octopusdeploy.slack.com/archives/CNHBHV2BX/p1707979639750569?thread_ts=1707909896.191589&cid=CNHBHV2BX) (internal link) for more details.